conda/recipes/numba-cuda/meta.yaml

```diff
@@ -28,6 +28,7 @@ requirements:
   run:
     - python
     - numba >=0.59.1
+    - cuda-bindings
```

Comment:
Just to confirm, for conda we're making this a required dependency whereas for `pip` we're making it an optional dependency. Is this intended?

Reply:
Since we're using the bindings by default, it doesn't feel perfect to me that they be optional anywhere - but based on https://github.com/NVIDIA/numba-cuda/pull/284#discussion_r2135880815 the goal IIUC is to be sensitive to what packages we pull in for a minimal installation.